utm.yaml

```diff
@@ -1122,12 +1122,17 @@ components:
             $ref: '#/components/schemas/Volume4D'
         state:
           $ref: '#/components/schemas/OperationState'
-        vlos:
-          description: If true, this Operation is ensuring deconfliction visually
-            and volumes of other Operations may overlap the volumes of this Operation.
-          type: boolean
-          default: false
+        priority:
+          $ref: '#/components/schemas/Priority'
 
+    Priority:
+      description: |-
+        Ordinal priority of the Operation, as defined by the regulator.  A lower-priority Operation
```

Comment:
We may want to be explicit that lower value means lower priority. Suggest adding sentence "Operations with lesser values are lower priority than all operations with greater values."

Reply:
Done. (note that when I say "Done", it means locally and will be reflected here when I push that commit after addressing the other comments)

Reply:
Ok. @BenjaminPelletier please resolve (I can't since I don't have write access).